src/snapred/backend/dao/request/FarmFreshIngredients.py

```diff
@@ -30,6 +30,7 @@ class FarmFreshIngredients(BaseModel):
     nBinsAcrossPeakWidth: int = Config["calibration.diffraction.nBinsAcrossPeakWidth"]
     peakIntensityThreshold: float = Config["calibration.diffraction.peakIntensityThreshold"]
     maxOffset: float = Config["calibration.diffraction.maximumOffset"]
+    dMin: Optional[float]
```

Comment:
This line shouldn't be necessary.  But if it is:
1. it should be called `crystalDMin`
2. delete `dBounds` and add another `crystalDMax` variable
3. this should use the same default values given below